src/com/google/enterprise/adaptor/WrapperAdaptor.java

```diff
@@ -407,6 +418,10 @@ public boolean isCrawlOnce() {
     public boolean isLock() {
       return lock;
     }
+
+    public Map<String, String> getParams() {
+      return params;
```

Comment:
This class should be marked VisibleForTesting.  It is used by CommandStreamParserTest and others that use TestHelper.  Basically the getters allow the tests to extract fields from the Response.